apps/web/components/booking/BookingListItem.tsx

```diff
@@ -403,16 +389,21 @@ const RecurringBookingsTooltip = ({
   recurringDates,
 }: RecurringBookingsTooltipProps) => {
   const { t } = useLocale();
+  const now = new Date();
 
   return (
     (booking.recurringBookings &&
       booking.eventType?.recurringEvent?.freq &&
-      (booking.listingStatus === "recurring" || booking.listingStatus === "cancelled") && (
+      (booking.listingStatus === "recurring" ||
+        booking.listingStatus === "unconfirmed" ||
+        booking.listingStatus === "cancelled") && (
```

Comment:
Should we use the conditions constants here?

Reply:
Different context as you surely noticed, and it was just used once there, so I found no need to create new constants.